src/tests/ftest/util/soak_utils.py

```diff
@@ -1199,6 +1202,7 @@ def create_app_cmdline(self, job_spec, pool, ppn, nodesperjob):
     app_params = os.path.join(os.sep, "run", job_spec, "*")
     app_cmd = os.path.expandvars(self.params.get("cmdline", app_params, default=None))
     mpi_module = self.params.get("module", app_params, self.mpi_module)
+    mpi_module_use = self.params.get("module_use", app_params, self.mpi_module_use)
```

Comment:
It is something else; The "apps" can run with any mpi such as IntelMPI if the apps were built with it and this can be different than what the rpms (ior, mpifileutils)  were built with.